pkg/reconciler/iploop.go

```diff
@@ -164,9 +165,12 @@ func (rl *ReconcileLooper) findClusterWideIPReservations(ctx context.Context) er
 
 	for _, clusterWideIPReservation := range clusterWideIPReservations {
 		ip := clusterWideIPReservation.GetName()
+		// De-normalize the IP
```

Comment:
May need to add comment why it is required (e.g. to prevent deletion?) because calling isPodAlive() with de-normalized IP seems to be unique in the code.

Reply:
Can you point at where we're normalizing the IP in this comment ?

Reply:
Rather than explain about "preventing deletion" (a specific bug) I am instead opting to explain why denormalization is correct. I have also pointed at where the IP is normalized in this comment:

 "In the UpdateOverlappingRangeAllocation function, the IP address is created with a "normalized" name to comply with the k8s api. We must denormalize here in order to properly look up the IP address in the regular format, which pods use."